redfish-core/lib/account_service.hpp

```diff
@@ -1374,6 +1560,40 @@ inline void updateUserProperties(std::shared_ptr<bmcweb::AsyncResp> asyncResp,
                     "xyz.openbmc_project.User.Attributes",
                     "UserLockedForFailedAttempt", std::variant<bool>{*locked});
             }
+            if (accountType)
+            {
+                crow::connections::systemBus->async_method_call(
+                    [accountType, asyncResp, dbusObjectPath,
+                     isUserItself](const boost::system::error_code ec,
+                                   std::variant<std::string>& userPrivilege) {
+                        if (ec)
+                        {
+                            BMCWEB_LOG_ERROR << "D-Bus responses error: " << ec;
+                            messages::internalError(asyncResp->res);
+                            return;
+                        }
+                        const std::string* userPrivPtr =
+                            std::get_if<std::string>(&userPrivilege);
+                        if (userPrivPtr == nullptr)
+                        {
+                            BMCWEB_LOG_ERROR << "UserPrivilege wasn't a "
+                                                "string";
+                            messages::internalError(asyncResp->res);
+                            return;
+                        }
+
+                        // check is user is admin user or not
+                        bool isAdminUser =
+                            (*userPrivPtr == "priv-admin") ? true : false;
```

Comment:
We typically don't use the ternary operator in bmcweb

Reply:
I tried to use the ?: operator once....
Can we use `bool isAdminUser = (*userPrivPtr == "priv-admin")`?
But we might not need this.  See my comment above where this is used.